app/assets/javascripts/govuk_publishing_components/modules.js

```diff
@@ -0,0 +1,61 @@
+;(function (global) {
```

Comment:
Were the files moved from the toolkit copied unmodified? Just want to check how thoroughly this needs to be reviewed.

Reply:
I copied them (both the file and the spec) 'as-is', for now, with the intention to update them in a future iteration (I'd really love to remove the jQuery dependency out of it and potentially improve it).